api/namex/utils/queue_util.py

```diff
@@ -0,0 +1,50 @@
+import json
+import time
+import uuid
+from datetime import datetime, timezone
+
+from config import Config
+from namex.services import queue
+
+
+def create_cloud_event_msg(msg_id, msg_type, source, time, identifier, json_data_body):  # pylint: disable=too-many-arguments # noqa E501
+    # industry standard arguments for this message
+    """Creates a payload for the cloud event messages."""
+    cloud_event_msg = {
+        'specversion': '1.x-wip',
+        'type': msg_type,
+        'source': source,
+        'id': msg_id,
+        'time': time,
+        'datacontenttype': 'application/json',
+        'identifier': identifier
+    }
+    if json_data_body:
+        cloud_event_msg['data'] = json_data_body
+
+    return cloud_event_msg
+
+def create_name_request_state_msg(nr_num, state_cd, old_state_cd):
+    """Builds a name request state message."""
+    return create_cloud_event_msg(msg_id=str(uuid.uuid4()),
+                                            msg_type='bc.registry.names.request',
```

Comment:
can we have the message type be something like `bc.registry.nr_state.changed`?  it will help the subscribers to a subject determine what type of message they are looking at if there happens to be different message types being published to a subject.

Reply:
👍

Reply:
bc.registry.names.events would be good, then it sets up for any event we want to listen for

Reply:
Changed to Thor's suggestion: bc.registry.names.events